dapr-spring/dapr-spring-boot-autoconfigure/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr.spring</groupId>
+    <artifactId>dapr-spring-parent</artifactId>
+    <version>0.13.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>dapr-spring-boot-autoconfigure</artifactId>
+  <name>dapr-spring-boot-autoconfigure</name>
+  <description>Dapr Spring Boot Autoconfigure</description>
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.dapr.spring</groupId>
+      <artifactId>dapr-spring-core</artifactId>
+      <version>${project.parent.version}</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>io.dapr.spring</groupId>
+      <artifactId>dapr-spring-data</artifactId>
+      <version>${project.parent.version}</version>
+      <optional>true</optional>
```

Comment:
@artur-ciocanu if these are optional, then, as a user we don't get these dependencies if we include the `dapr-spring-boot-starter` which means that we need to separately add this dependencies manually.. was there a reason why this needs to be optional?